CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# How to contribute to FreeOrion
+
+As open source project FreeOrion both relies on contributions from both long
+time developers and one time occasion contributors.  This guide should help you
+to create contributions that can be easily integrated into the FreeOrion
+project.
+
+
+## I found an error in the game or I want to file a bug report
+
+So you stumbled over a reproducible error in FreeOrion?  Then you can report it
+into the [FreeOrion Issues].  But please check before reporting the error if
+someone else has already posted it.  When creating a new issue you will see a
+template of required data.  Please fill those template with all the information
+you know.
+
```

Comment:
So you stumbled on a reproducible error in FreeOrion?  You can report it
in [FreeOrion Issues].  Please check before reporting the error to see if someone else has already posted it.  When creating a new issue you will see a template of required data.  Please fill in the template with all the information that
you know.


over->on.   This is stylistic not grammatical change.
Remove "Then".  Also a style comment.  Starting the sentence with "Then" sounds like a programmer turn of phrase.
Remove "But"  Also style.
Add "to see"
"those" -> "in the"
"you know" -> "that you know"